File: dart/Hashing.dart
void main() {
  String str = "Banana";
  String newStr = str.toLowerCase();

  Set<String> ch = {};

  for (var char in newStr.split('')) {
    if (ch.contains(char)) {
      ch.remove(char);
    } else {
      ch.add(char);
    }
  }
  print(ch.join(''));
}

File: java/Hashing.java
import java.util.*;
import java.util.stream.Collectors;

class Hashing {
    public static void main(String[] args) {
        String str = "YASHASHRI";
        String newStr = str.toLowerCase();

        Set<Character> ch = new HashSet<>();

        for (char c : newStr.toCharArray()){
            if(ch.contains(c)){
                ch.remove(c);
            }else{
                ch.add(c);
            }
        }

         String resultString = ch.stream()
                                 .map(String::valueOf)
                                 .collect(Collectors.joining(""));

                                 System.out.println(resultString);


    }
}
